proto/spire/api/agent/logger/v1/logger.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package spire.api.agent.logger.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/agent/logger/v1;loggerv1";
+
+import "spire/api/types/logger.proto";
+
+service Logger {
+
+    // Gets the logger level.
+    //
+    // The caller must be local or present an admin X509-SVID.
```

Comment:
The agent doesn't have the same authorization mechanisms as the server. This API will only be available over UDS. Probably appropriate to just remove this sentence (for all 3 RPCs) since anybody with access to the private admin socket has permissions to execute any function.

Reply:
I'll alter all of the agent commentary to say something about it being serviced on the admin socket.